tools/postReleaseScripts/publishGHRelease.ps1

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+#requires -Version 6.0
+
+param(
+    [Parameter(Mandatory)]
+    [semver]
+    $Version,
+
+    [Parameter(Mandatory)]
+    [string]
+    $GitHubToken,
+
+    [Parameter(Mandatory)]
+    [string]
+    $Repository,
+
+    [Parameter()]
+    [string]
+    $TargetFork = 'PowerShell',
+
+    [Parameter()]
+    [string]
+    $ChangelogPath = "$PSScriptRoot/../../CHANGELOG.md",
+
+    [Parameter()]
+    [string[]]
+    $AssetPath
+)
+
+Import-Module "$PSScriptRoot/../GitHubTools.psm1" -Force
+
+function GetDescriptionFromChangelog
+{
+    param(
+        [Parameter(Mandatory)]
+        [string]
+        $ChangelogPath
+    )
+
+    $lines = Get-Content -Path $ChangelogPath
+    $sb = [System.Text.StringBuilder]::new($lines[2])
+    for ($i = 3; -not $lines[$i].StartsWith('## '); $i++)
```

Comment:
nit a small comment since there are some numbers that are unknown.

I'm assuming this just grabs the right chunk of the changelog?

Reply:
Yeah -- I'll add a comment